src/libgru/makefile.in

```diff
@@ -15,15 +15,15 @@ MKDIR_P                       = @MKDIR_P@
 .PHONY                        : all install install-strip clean distclean
 
 all                           : $(PROGNAME)
-install                       : | $(bindir)
-	$(INSTALL) -p $(PROGNAME) $(bindir)
+install                       : | $(DESTDIR)/$(bindir)
+	$(INSTALL) -p $(PROGNAME) $(DESTDIR)$(bindir)
 install-strip                 :
-	$(MAKE) INSTALL="$(INSTALL) -s" install
+	$(MAKE) INSTALL="$(INSTALL) -s" DESTDIR=$(DESTDIR) install
 clean                         :
-	rm -f $(PROGNAME)
+	rm -f $(DESTDIR)$(PROGNAME)
 distclean                     :
 	rm -f makefile
 $(PROGNAME)                   : $(SRC)
 	$(CC) $(CPPFLAGS) $(CFLAGS) $(LDFLAGS) $^ $(LDLIBS) -o $@
-$(bindir)                     :
-	$(MKDIR_P) $(bindir)
+$(DESTDIR)/$(bindir)                     :
```

Comment:
Nitpicking; the `:` is misaligned and the file is missing a newline at the end.

Reply:
Done.